components/interactive_dialog/dialog_element/dialog_element.jsx

```diff
@@ -50,10 +51,17 @@ export default class DialogElement extends React.PureComponent {
             } else if (props.options) {
                 this.providers = [new MenuActionProvider(props.options)];
             }
+
+            if (props.value && props.options) {
+                const defaultOption = props.options.find(
+                    (option) => option.value === props.value
+                );
+                defaultText = defaultOption.text || '';
```

Comment:
In the case where the default isn't found in `options` this could throw. Might want to do a null check on `defaultOption`